manifests/two_pods_talking/route.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: test-pod
+spec:
+  to:
+    kind: Service
+    name: test-pod
```

Comment:
should this be `test-pod`? since that's what's the service is called in `manifests/two_pods_talking/service.yaml`

Reply:
This resource says "create a Route named `test-pod` and connect it to a Service named `test-pod`". I may not have entirely understood your question, but this looks okay to me.

Reply:
Oh, wait, I see what you're saying. This resource is actually erroneous: we don't use the route for the `two_pods_talking` test. I'll remove it.